components/raftstore/src/store/peer.rs

```diff
@@ -1721,24 +1739,32 @@ where
                     self.raft_group.skip_bcast_commit(true);
                     self.last_urgent_proposal_idx = u64::MAX;
                 }
+                let current_term = self.term();
+                let cbs = committed_entries
+                    .iter()
+                    .filter_map(|e| {
```

Comment:
How about using `retain`?

Reply:
The purpose is to find `Proposal` for each committed entry, not changing `committed_entries`